public/js/controllers/attackcost_controller.js

```diff
@@ -3,6 +3,7 @@ import TurboQuery from '../helpers/turbolinks_helper'
 import { getDefault } from '../helpers/module_helper'
 import globalEventBus from '../services/event_bus_service'
 import dompurify from 'dompurify'
+import _ from 'lodash-es'
```

Comment:
I have yet to find a case where I felt `lodash` was actually an improvement over vanilla js. I'm sure there are some, I just haven't seen one.